appservice/src/createAppService/AppServicePlanListStep.ts

```diff
@@ -39,6 +39,7 @@ export class AppServicePlanListStep extends AzureWizardPromptStep<IAppServiceWiz
         const options: IAzureQuickPickOptions = { placeHolder: localize('selectPlan', 'Select a {0} App Service plan.', getWebsiteOSDisplayName(nonNullProp(wizardContext, 'newSiteOS'))), id: `AppServicePlanListStep/${wizardContext.subscriptionId}` };
         wizardContext.plan = (await ext.ui.showQuickPick(this.getQuickPicks(wizardContext), options)).data;
 
+        wizardContext.telemetry.properties.newPlan = String(!wizardContext.plan);
```

Comment:
Is there any benefit to this over `toString()`?

Reply:
`String(value)` works even if value is null/undefined so as a general rule I do it more often than `value.toString()`